bin/rspec

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env ruby
+
+spec_path = File.expand_path(File.join(File.dirname(__FILE__), "../spec"))
+
+$LOAD_PATH << File.expand_path(File.join("__FILE__", "..", "lib"))
+$LOAD_PATH << spec_path
```

Comment:
$LOAD_PATH <<  File.expand_path(File.join(File.dirname(__FILE__), "../spec"))
spec_path is useless